core/cmd/local_client_test.go

```diff
@@ -86,7 +86,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 	awaiter := cltest.NewAwaiter()
 	go func() {
 		assert.NoError(t, cmd.NewApp(&client).
-			Run([]string{"", "node", "start", "-debug", "-password", "../internal/fixtures/correct_password.txt"}))
+			Run([]string{"", "-password", "../internal/fixtures/correct_password.txt", "node", "start", "-debug"}))
```

Comment:
Unfortunately, moving the flags to the root level means they must be specified there as well, requiring node ops to make a change when upgrading. :thinking: :balance_scale:

Reply:
undone!